doc/source/reference/c-api/array.rst

```diff
@@ -2303,21 +2303,29 @@ Item selection and manipulation
 
 .. c:function:: PyObject* PyArray_Sort(PyArrayObject* self, int axis, NPY_SORTKIND kind)
 
-    Equivalent to :meth:`ndarray.sort<numpy.ndarray.sort>` (*self*, *axis*, *kind*).
-    Return an array with the items of *self* sorted along *axis*. The array
-    is sorted using the algorithm denoted by *kind*, which is an integer/enum pointing
-    to the type of sorting algorithms used.
-
-.. c:function:: PyObject* PyArray_ArgSort(PyArrayObject* self, int axis)
-
-    Equivalent to :meth:`ndarray.argsort<numpy.ndarray.argsort>` (*self*, *axis*).
-    Return an array of indices such that selection of these indices
-    along the given ``axis`` would return a sorted version of *self*. If *self* ->descr
-    is a data-type with fields defined, then self->descr->names is used
-    to determine the sort order. A comparison where the first field is equal
-    will use the second field and so on. To alter the sort order of a
-    structured array, create a new data-type with a different order of names
-    and construct a view of the array with that new data-type.
+    Equivalent to :meth:`ndarray.sort<numpy.ndarray.sort>` (*self*, *axis*,
```

Comment:
This was already not quite true any more, and will soon be even less so; I'd tend to start with what the function actually does (so, "Return an array ... " and then at the very end say something like
```
This function is the implementation for :meth:`ndarray.sort<numpy.ndarray.sort>`,
though with slightly different meaning of ``kind`` -- see ``NPY_SORTKIND`` below.
```

(and similar for argsort)